proxy/proxy.go

```diff
@@ -172,13 +173,18 @@ func (p *T) stopAdmin() {
 //
 // Errors usually indicate a catastrophic failure of the Kafka cluster, or
 // missing topic if there cluster is not configured to auto create topics.
-func (p *T) Produce(topic string, key, message sarama.Encoder) (*sarama.ProducerMessage, error) {
+func (p *T) Produce(topic string, key, message sarama.Encoder, headers []sarama.RecordHeader) (*sarama.ProducerMessage, error) {
+	if len(headers) > 0 &&
+		!p.cfg.Kafka.Version.IsAtLeast(sarama.V0_11_0_0) {
```

Comment:
Could you please keep it on the same line as `if`. There is no 80 chars per line limitation.

Reply:
No problem, fixed.